src/Application/FacetConfig.php

```diff
@@ -13,7 +13,7 @@ class FacetConfig {
 	 * @param array<string, mixed> $typeSpecificConfig
 	 */
 	public function __construct(
-		public readonly ItemId $itemTypeId,
+		public readonly ItemId $itemType,
 		public readonly PropertyId $propertyId,
```

Comment:
I assume we'll do a similar refactor to change `$propertyId` to `$property` here?

Reply:
Unclear that would be better

`itemTypeId vs itemType` and `propertyId vs property` are not equivalent